tools/gce-quota-sync/sample-deployment.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: apps/v1
```

Comment:
Have you considered making this a simple Helm chart instead? (https://docs.helm.sh/chart_template_guide/)
Even in a small deployment, it can be hard to determine what values are variables that should be altered by the end user.

Reply:
The deployment is so simple that a Helm chart looks a bit overkill (and not everybody uses Helm). One other option might be to remove the sample deployment file, and show the environment block in the README.

Reply:
Yeah, removing the deployment looks to be the way to go. You don't have to go all-in with Helm+Tiller, you can just do standalone deployments, but they don't advertise that fact very extensively.